src/brogue/RogueMain.c

```diff
@@ -218,13 +218,15 @@ void initializeRogue(unsigned long seed) {
     generateFontFiles();
 #endif
 
-    playingback = rogue.playbackMode; // the only three animals that need to go on the ark
+    playingback = rogue.playbackMode; // the only four animals that need to go on the ark
     playbackPaused = rogue.playbackPaused;
     playbackFF = rogue.playbackFastForward;
+    serverMode = rogue.serverMode;
     memset((void *) &rogue, 0, sizeof( playerCharacter )); // the flood
     rogue.playbackMode = playingback;
     rogue.playbackPaused = playbackPaused;
     rogue.playbackFastForward = playbackFF;
+    rogue.serverMode = serverMode;
```

Comment:
Just thinking out loud, do you think this might be more suitable to be a global variable? I normally think of `rogue` as containing information pertaining to the gameplay or "game view" and not the broader behaviour. Would save juggling it around this memset too.

Reply:
Sure, the origin version had it as an extern global like the original noMenu, I think I prefer this slightly. I'll put it back.

Reply:
FYI in Rogue.h:

```c
typedef struct playerCharacter {
    ....
    // What do you want to do, player -- play, play with seed, resume, recording, high scores or quit?
    enum NGCommands nextGame;
    char nextGamePath[BROGUE_FILENAME_MAX];
    unsigned long nextGameSeed;
```